vector/build.gradle

```diff
@@ -447,6 +447,8 @@ dependencies {
     implementation 'com.vanniktech:emoji-material:0.7.0'
     implementation 'com.vanniktech:emoji-google:0.7.0'
 
+    implementation 'im.dlg:android-dialer:1.2.5'
```

Comment:
Is there an entry for it in open_source_licences.html?

Reply:
Nice catch. But it'll probably be removed by a custom design. Still, I add it.